examples/layer-browser/src/utils/grid-aggregator.js

```diff
@@ -27,7 +27,7 @@ export function pointsToWorldGrid(points, worldUnitSize) {
     return accu;
   }, {});
 
-  const maxHeight = Math.max.apply(null, Object.values(gridHash));
+  const maxHeight = Math.max.apply(null, Object.keys(gridHash).map(k => gridHash[k]));
```

Comment:
Logic looks fine.

A concern could be if gridHash could get big? We are generating two unnecessary temporary arrays here to be able to apply `Math.max`. If those arrays could be very large we may want to avoid this.

if instead iterated over the keys using `for in` we could do this without temporaries.